components/nimbus/src/stateless/cirrus_client.rs

```diff
@@ -76,20 +77,25 @@ pub struct CirrusMutableState {
     experiments: Vec<Experiment>,
 }
 
-#[derive(Default)]
 pub struct CirrusClient {
     app_context: AppContext,
     coenrolling_feature_ids: Vec<String>,
     state: Mutex<CirrusMutableState>,
+    metrics_handler: Arc<Box<dyn MetricsHandler>>,
```

Comment:
Hmm. This is going to be a breaking change for cirrus over in mozilla/experimenter. I wonder if we should migrate that python (or a portion of it) and its tests over here.

Reply:
That's very possible — we could likely do something similar to what we have set up with the Kotlin/Swift